Lib/fontParts/base/font.py

```diff
@@ -1115,78 +1517,106 @@ def __setitem__(self, name, glyph):
 
     # order
 
-    glyphOrder = dynamicProperty(
+    glyphOrder: dynamicProperty = dynamicProperty(
         "base_glyphOrder",
-        """
-        The preferred order of the glyphs in the font.
+        """Get or set the order of the glyphs in the font.
+
+        The value must be a :class:`list` of glyph names reflecting the
+        desired order of the font's :class:`BaseGlyph` objects.
+
+        :return: A :class:`list` of glyph names in their defined order.
+
+        Example::
 
             >>> font.glyphOrder
             ["C", "B", "A"]
             >>> font.glyphOrder = ["A", "B", "C"]
+
         """
     )
 
-    def _get_base_glyphOrder(self):
+    def _get_base_glyphOrder(self) -> List[str]:
```

Comment:
Does type need to be Tuple here or is List good?

Reply:
No, should be `tuple`. All of these methods were wrongly typed and documented, but are now fixed.